cloudflare/resource_cloudflare_worker_script.go

```diff
@@ -50,6 +52,20 @@ var secretTextBindingResource = &schema.Resource{
 	},
 }
 
+var webAssemblyBindingResource = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"name": {
+			Type:     schema.TypeString,
+			Required: true,
+		},
+		"base64": {
```

Comment:
I find the schema name here a little lacking; what are your thoughts on using something more aligned with the existing schema fields but still convey it's base64 encoded, like `base64_encoded_text`? We could also reuse `text` here and note in the documentation it is the bas64 encoded value?

Reply:
Hm, I'm not particularly happy with the notion of "text" either, what's actually supposed to be here is a wasm module, which is binary data. `module` probably works best, changing it.

Reply:
How about `data`? `module` leaves some ambiguity for a novice like me.

Reply:
It feels like `data` would be more ambiguous than `module`, the latter being defined in spec to be this exact thing

Reply:
Is this still a point of contention? I can deal with it being data `data`

Reply:
No point of contention from me, just want this make sense to those that are going to be using it. I have little experience here so happy to leave as-is for your wording.